src/Network/HttpClient.php

```diff
@@ -102,26 +83,14 @@ private function getClient()
      * Sends request to Google Analytics.
      *
      * @param string $url
-     * @param SingleParameter[] $singleParameters
-     * @param CompoundParameterCollection[] $compoundParameters
      * @param boolean $nonBlocking
      * @return AnalyticsResponse
      */
-    public function post($url, array $singleParameters, array $compoundParameters, $nonBlocking = false)
+    public function post($url, $nonBlocking = false)
```

Comment:
@venyii but the HttpClient class is not meant to be used directly... I mean, the method signature changed and its BC Break if you were using it directly, but that's not how its meant to be used...

Are you using it HttpClient directly in your code?